torch/csrc/jit/tensorexpr/scripts/bisect.py

```diff
@@ -10,6 +10,7 @@ def test(cmd, limit):
         shell=True,
         capture_output=True,
         encoding="utf-8",
+        check=True,
```

Comment:
It should be false there, shouldn't it (as `p.error_code` is checked few lines below)
```suggestion
        check=False,
```